src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  flex-direction: row;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding-right: 50px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  height: 15px;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  margin: 0 10px;
+  padding: 22.5px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  background-color: #00acdc;
+  top: 56px;
+  right: 650px;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+}
+
+.first {
+  margin-left: 0;
+}
+
+.last {
```

Comment:
this should be also removed, try comined use margin for link as :

```
.nav__link:not(:last-child) {
  margin-right: 20px;
}
```

Reply:
After trying this I can only say: I fucked up. 
Can't make it to work with :last-child or :first-child - witch means i really fucked something up. 
It's working my way but it looks terrible.